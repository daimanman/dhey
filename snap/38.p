{"C":1,"Z":"","N":1,"Remark":"压力测试","Url":"http://localhost:9188/go1","Method":"POST","TaskId":38,"Type":"FIXED","FileId":"4","Status":0,"Err":"","QH":{},"P":{"info":"TEST","name":"DXM"},"Payload":""}