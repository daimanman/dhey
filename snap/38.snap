{"AvgTotal":0.0130341,"Fastest":0.0130341,"Slowest":0.0130341,"Average":0.0130341,"Rps":76.72182966219378,"AvgConn":0.0050128,"AvgDNS":0.0040096,"AvgReq":0,"AvgRes":0,"AvgDelay":0.0030078,"ConnMax":0.0040096,"ConnMin":0.0040096,"DnsMax":0.0040096,"DnsMin":0.0040096,"ReqMin":0,"ReqMax":0,"ResMin":0,"ResMax":0,"DelayMax":0.0030078,"DelayMin":0.0030078,"Lats":[0.0130341],"ConnLats":[0.0040096],"DnsLats":[0.0040096],"ReqLats":[0],"ResLats":[0],"DelayLats":[0.0030078],"Total":13034100,"ErrorDist":{},"StatusCodeDist":{"200":1},"SizeTotal":2,"SizeReq":2,"NumRes":1,"LatencyDistribution":[{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.0130341,"Count":1,"Frequentcy":1},{"Mark":0.0130341,"Count":0,"Frequentcy":0},{"Mark":0.0130341,"Count":0,"Frequentcy":0},{"Mark":0.0130341,"Count":0,"Frequentcy":0},{"Mark":0.0130341,"Count":0,"Frequentcy":0},{"Mark":0.0130341,"Count":0,"Frequentcy":0},{"Mark":0.0130341,"Count":0,"Frequentcy":0},{"Mark":0.0130341,"Count":0,"Frequentcy":0},{"Mark":0.0130341,"Count":0,"Frequentcy":0},{"Mark":0.0130341,"Count":0,"Frequentcy":0},{"Mark":0.0130341,"Count":0,"Frequentcy":0}],"TaskId":38,"Duration":0.0130341}